src/libs/rover/scheduler.cpp

```diff
@@ -651,31 +658,25 @@ void Scheduler<FloatType>::to_blueprint(conduit::Node &dataset)
       conduit::Node &n_int = dataset["fields/intensities"];
       n_int["topology"] = topo_name;
       n_int["association"] = "element";
+      n_int["units"] = "intensity units";
       vtkm::cont::ArrayHandle<FloatType> ints = m_result.flatten_intensities();
       FloatType *ints_buffer = get_vtkm_ptr(ints);
       // can't set external since this goes out of scope
       n_int["values"].set(ints_buffer, ints.GetNumberOfValues());
-      n_int["shape"].set(shape);
       n_int["strides"].set(strides);
```

Comment:
I'm not convinced that the data is strided the way we want it to be strided.

We will need to investigate/discuss further. Perhaps we need to create some really low resolution examples (like 4x5 images) to understand how this differs from VisIt.

I think a good TODO would be to add a test that runs the Filter at a lower resolution (than 1024x1024). It would be good to know how much control we really have over the camera. What is default; what is able to be specified? We can do a deeper dive into the camera in a follow-on PR.

Reply:
We probably want most tests to be at a lower resolution than 1024x1024. I think the defaults should be changed too to match VisIt.